block/stream/output/smart_contract_service.proto

```diff
@@ -132,10 +132,23 @@ message EthereumOutput {
      */
     repeated proto.TransactionSidecarRecord sidecars = 1;
 
-    /**
-     * An ethereum hash value.
-     * <p>
-     * This SHALL be a keccak256 hash of the ethereumData.
-     */
     bytes ethereum_hash = 2;
+
+    oneof eth_result {
+        /**
+         * A result for an Ethereum Transaction executed as a call.
+         * <p>
+         * This field SHALL contain all of the data produced by the contract
+         * call transaction as well as basic accounting results.
+         */
+        proto.ContractFunctionResult ethereum_call_result = 3;
+
+        /**
+         * A result for an Ethereum Transaction executed as a create.
+         * <p>
+         * This field SHALL contain all of the data produced by the contract
+         * create transaction as well as basic accounting results.
+         */
+        proto.ContractFunctionResult ethereum_create_result = 4;
+    }
```

Comment:
We already have the `CallContractOutput` and `CreateContractOutput` items, why are we adding the same data again here? A "canonical" Block Stream would simply follow this `EthereumOutput` with one of those items if necessary.

Reply:
Ah that is an interesting idea @jsync-swirlds  and would work perfectly well. Created https://github.com/hashgraph/hedera-services/issues/15619 and self-assigned.